### PR TITLE
fix EMAN2-2.21a (WIP)

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)
depends on PR ~~#6531~~

### CODE REVIEW

**Comment by boegel** [comment]
@smoors What problem are you fixing here exactly, which error messages were you getting?

**Comment by smoors** [comment]
@boegel 
the GUI works, but calculations do not work because they need Pydusa, which I forgot to add.
furthermore the EMAN2 software is actually a combination of EMAN2 and SPHIRE.
the SPHIRE GUI needs PyOpenGL and matplotlib.

File: easybuild/easyconfigs/p/Pydusa/Pydusa-20180515_numpy_header_path.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
+diff -ur pydusa-20180515.orig/configure pydusa-20180515/configure
+--- pydusa-20180515.orig/configure	2018-05-15 17:01:43.000000000 +0200
++++ pydusa-20180515/configure	2018-07-02 11:19:06.219901276 +0200
+@@ -5097,6 +5097,8 @@
+    PY_HEADER_NUMPY="-I${PY_PREFIX}/Extras/lib/python/numpy/core/include"
+ elif test -d ${PY_PREFIX}/lib/python$PY_VERSION/site-packages/numpy/core/include; then 
+    PY_HEADER_NUMPY="-I${PY_PREFIX}/lib/python$PY_VERSION/site-packages/numpy/core/include"   
++elif test -d $PY_HEADER_NUMPY; then
++   PY_HEADER_NUMPY="-I$PY_HEADER_NUMPY"
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pydusa/Pydusa-20180515_numpy_header_path.patch]: @smoors Is this patch scrictly required? Why not use `$PY_PREFIX` and `$PY_VERSION` instead?

**Comment by smoors** [comment]
there are still some SPHIRE components not working with this version

File: easybuild/easyconfigs/p/Pydusa/Pydusa-20180515_numpy_header_path.patch (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/Pydusa/Pydusa-20180515_numpy_header_path.patch]: Sorry, I misunderstood this I guess, this is about letting the `configure` script pick up on `PY_HEADER_NUMPY` itself...

**Comment by timeu** [comment]
@smoors I am curious which SPHIRE components are not working ?

**Comment by smoors** [comment]
@timeu the sxmeridien.py program crashes (and possibly others). I think it was an MPI issue.
I am not working on it currently as it is not an urgent issue for us anymore, so feel free to pick it up if you like.

**Comment by timeu** [comment]
@smoors Ok I see, thanks for the info. 
We had a customer asking for it and I installed following the sphire guide (http://sphire.mpg.de/wiki/doku.php?id=downloads:sphire_1_1&redirect=1#source_code_installation_within_an_existing_anaconda_installation). I couldn't use the system (easybuild based) OpenMPI installation as it uses `--enable-dlopen` which Pydusa seems to have  issues with. So I had to build an OpenMPI version in the conda environment and rebuild Pydusa against it. I am now for the customer's feedback and if it works I will try to continue to get this solved with EasyConfig based on your PR.

**Pull request closed without merging**